Rules.md

```diff
@@ -178,67 +178,60 @@ Sea-sea movement is not possible for a unit if any land region at the strait is
 
 ## Battles
 
-When a move proceeds into an enemy region, a battle is initiated with all moved units.
+When a move proceeds into an enemy region, a battle is initiated between all units moved into the
+region versus the enemy units already in that region.
 
 ### Battle mechanic
 
-A battle has one or more rounds. Every round has following parts:
+A battle has the following phases:
 
 1. **Artillery strike:**
-    * Every healthy attacking artillery kills 1 unit (\*)
+    * Every attacking artillery kills 1 unit (\*).
 1. **Rolling dice:**
-    * The attacker gets a dice for every healthy attacking **infantry unit**
-    * The defender gets a dice for every healthy defending **unit** (infantry + artillery)
+    * The attacker gets a dice for every attacking **infantry unit**
+    * The defender gets a dice for every surviving defending **unit** (infantry + artillery)
```

Comment:
I added that here actually independent of this PR. It was intended to clarify that you don't get dice for units that were killed by the artillery strike.

Didn't really clarify much I guess :)